codersdk/provisionerdaemons.go

```diff
@@ -39,7 +38,7 @@ const (
 type ProvisionerDaemon struct {
 	ID           uuid.UUID         `json:"id" format:"uuid"`
 	CreatedAt    time.Time         `json:"created_at" format:"date-time"`
-	UpdatedAt    sql.NullTime      `json:"updated_at" format:"date-time"`
+	UpdatedAt    NullTime          `json:"updated_at" format:"date-time"`
```

Comment:
I think we still need `,omitempty` with `codersdk.NullTime` to get the right kind of docs generated for this. This can be either `null` or `string`.

```suggestion
	UpdatedAt    NullTime          `json:"updated_at,omitempty" format:"date-time"`
```